java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -112,9 +113,11 @@ private static String runCommand(String... command) {
             throw new WebDriverException("Unsuccessful command executed: " + Arrays.toString(command) +
                     "\n" + output);
         }
-
-        String[] lines = output.split("\n");
-        return lines[lines.length -1].replace(INFO, "").trim();
+        SeleniumManagerJsonOutput jsonOutput = new Json().toType(output,
+                SeleniumManagerJsonOutput.class);
+        jsonOutput.logs.stream().filter(log -> log.level.equalsIgnoreCase(WARN))
+                .forEach(log -> LOG.warning(log.message));
```

Comment:
@bonigarcia / @diemol can we log the debug information to `LOG.FINE` here? Is this the right place?

Reply:
Yes, we can do that.